tests/tests/Data/RawCodeCoverageDataTest.php

```diff
@@ -362,6 +364,7 @@ public function testHeavyIndentationIsHandledCorrectly(): void
                 117,
                 120,
                 123,
+                125, // This shouldn't be marked as LoC, but it's high unlikely to happen IRL to have $var = []; on multiple lines
```

Comment:
I've thoroughly checked each test that needed to be changed and everything is an edge case now fixed, except this one.

`$var = [];` on multiple lines will be marked on `[]` line too with this PR, which is wrong, but it's unlikely to happen on real code.

As far as I can tell, it's necessary to introduce this little bug to fix much more common ones.

Reply:
Makes sense to me.